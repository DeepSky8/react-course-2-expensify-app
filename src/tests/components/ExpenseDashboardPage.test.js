import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('should display Expense Dashboard with no items', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<ExpenseDashboardPage />)
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})