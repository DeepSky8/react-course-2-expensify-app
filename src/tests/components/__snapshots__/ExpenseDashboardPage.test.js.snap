// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display Expense Dashboard with no items 1`] = `
<div>
  <Memo(Connect(ExpensesSummary)) />
  <Memo(Connect(ExpenseListFilters)) />
  <Memo(Connect(ExpenseList)) />
</div>
`;
